.travis.yml

```diff
@@ -5,20 +5,11 @@ services:
 - docker
 - mysql
 go:
-- 1.8.1
-
-install:
-# This script is used by the Travis build to install a cookie for
-# go.googlesource.com so rate limits are higher when using `go get` to fetch
-# packages that live there.
-# See: https://github.com/golang/go/issues/12933
-- bash scripts/gogetcookie.sh
-- go get github.com/kardianos/govendor
+- 1.10.x
```

Comment:
Should you just go all the way to 1.11?

Reply:
Travis only supports to `1.10.x` out of the box so I got lazy and did a `go mod vendor` locally. 😂